.clangd

```diff
@@ -0,0 +1,3 @@
+Style:
+  AngledHeaders:
+    - ".*" # Headers in third-party directories
```

Comment:
We include private headers with double quotes (e.g. headers in src/core/)

Reply:
@avikivity thanks for the note - it's rare enough I've forgotten!

I've adjusted the regex to apply only to seastar public headers and tested it out on a few use cases and it looks good.